src/legacy/core_plugins/vis_type_timeseries/public/components/aggs/math.js

```diff
@@ -121,9 +121,9 @@ export class MathAgg extends Component {
                 <FormattedMessage
                   id="visTypeTimeseries.math.expressionDescription"
                   defaultMessage="This field uses basic math expressions (see {link}) - Variables are keys on the {params} object,
-                i.e. {paramsName} To access all the data use {paramsValues} for an array of the values and {paramsTimestamps} for
+                i.e. {paramsName}. To access all the data use {paramsValues} for an array of the values and {paramsTimestamps} for
                 an array of the timestamps. {paramsTimestamp} is available for the current bucket's timestamp,
-                {paramsIndex} is available for the current bucket's index, and {paramsInterval}s available for
+                {paramsIndex} is available for the current bucket's index, and {paramsInterval} is available for
```

Comment:
When we change the message, but not the ID, of an internationalized string, we need to remove the message from the translation `.json` files as well.